resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/VolcanoTestsSuite.scala

```diff
@@ -402,8 +410,8 @@ private[spark] trait VolcanoTestsSuite extends BeforeAndAfterEach { k8sSuite: Ku
         m += (p -> Instant.parse(scheduledTime))
       }
       // high --> medium --> low
-      assert(m("high").isBefore(m("medium")))
-      assert(m("medium").isBefore(m("low")))
+      assert(m("high").isBefore(m("medium")) || m("high").equals(m("medium")))
```

Comment:
then we can remove equal because the priority driver can be completed in serial due to queue cap.

Reply:
Ah, I was curious about that. Thanks!